java/RSPEC-131.java

```diff
@@ -0,0 +1,11 @@
+class SwitchNoDefault {
+    void test() {
+        switch (myVariable) {
```

Comment:
RSPEC-131: Switch without default case
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._